docs/usage/configuration-options.md

```diff
@@ -180,7 +180,7 @@ This field is combined with `branchPrefix` and `additionalBranchPrefix` to form
 
 ## bumpVersion
 
-Currently this setting supports `npm` only, so raise a feature request if you have a use for it with other package managers. It's purpose is if you want Renovate to update the `version` field within your file's `package.json` any time it updates dependencies within. Usually this is for automatic release purposes, so that you don't need to add another step after Renovate before you can release a new version.
+Currently this setting supports `npm` and `gradle` only, so raise a feature request if you have a use for it with other package managers. It's purpose is if you want Renovate to update the `version` field within your file's `package.json` any time it updates dependencies within. Usually this is for automatic release purposes, so that you don't need to add another step after Renovate before you can release a new version.
```

Comment:
```suggestion
Currently this setting supports `npm` and `gradle` only, so raise a feature request if you have a use for it with other package managers.
Its purpose is if you want Renovate to update the `version` field within your file's `package.json` any time it updates dependencies within.
Usually this is for automatic release purposes, so that you don't need to add another step after Renovate before you can release a new version.
```

This suggestion ensures this text conforms to the one sentence per line style.
This suggestion also fixes a small `it's` vs `its` problem.